pkg/processor/metricfrequencyprocessor/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
newline :see_no_evil:

Reply:
This file is copied from other processors. They all omit this newline.